packages/smart-contracts/src/lib/artifacts/BatchConversionPayments/index.ts

```diff
@@ -13,51 +13,47 @@ export const batchConversionPaymentsArtifact = new ContractArtifact<BatchConvers
           address: '0x2e335F247E91caa168c64b63104C4475b2af3942',
           creationBlockNumber: 0,
         },
-        rinkeby: {
-          address: '0x42c466ccF141A8EBE0Aa2ECEF06A4f4BE9a9B526',
-          creationBlockNumber: 11338182,
-        },
         goerli: {
-          address: '0x42c466ccF141A8EBE0Aa2ECEF06A4f4BE9a9B526',
-          creationBlockNumber: 7546738,
+          address: '0x0fD49d410fB50Dc1f308975E5D19B5c4551B0041',
```

Comment:
why the change of address, and not a new version? What will happen to payments already made?

Reply:
As discussed, batch proxy addresses are only used by the payment processor.
Keeping the old addresses (and json) is not needed.